### PR TITLE
Add support for snowflake.yml in procedures

### PR DESCRIPTION
**Pull request by sfc-gh-turbaszek**
### Pre-review checklist
   * [ ] I've confirmed that instructions included in README.md are still correct after my changes in the codebase.
   * [ ] I've added new automated tests to verify correctness of my new code.
   * [ ] I've confirmed that my changes are up-to-date with the target branch.
   * [ ] I've described my changes in the release notes.
   * [ ] I've described my changes in the section below.

### Changes description
...

### CODE REVIEW

File: src/snowcli/cli/snowpark/common.py [hunk 1]
```diff
@@ -189,3 +180,11 @@ def describe(self, identifier: str) -> SnowflakeCursor:
...
     def artifact_stage_path(identifier: str):
         return generate_deploy_stage_name(identifier).lower()
+
+
+def build_udf_sproc_identifier(udf_sproc_dict):
+    arguments = ", ".join(
+        (f"{arg['name']} {arg['type']}" for arg in udf_sproc_dict["signature"])
+    )
+    identifier = f"{udf_sproc_dict['name']}({arguments})"
+    return identifier
```
Comment (sfc-gh-astus) [inline src/snowcli/cli/snowpark/common.py:190]:
```suggestion
    return f"{udf_sproc_dict['name']}({arguments})"
```

File: src/snowcli/cli/snowpark/procedure/commands.py [hunk 2]
```diff
@@ -101,115 +108,143 @@
...
+    artifact_path: Path,
+    handler: str,
+    identifier: str,
 ):
-    stage_manager = StageManager()
-    stage_manager.create(
-        stage_name=DEPLOYMENT_STAGE, comment="deployments managed by snowcli"
+    signature_start_index = identifier.index("(")
+    name = identifier[0:signature_start_index]
+    signature = identifier[signature_start_index:]
```
Comment (sfc-gh-astus) [inline src/snowcli/cli/snowpark/procedure/commands.py:119]: Good candidate for util 😄

**Comment by sfc-gh-pjob** [comment]
looks good to me

**Review by sfc-gh-pjob — DISMISSED** [review]

**Comment by sfc-gh-pjob** [comment]
Please add release notes.

**Review by sfc-gh-pjob — APPROVED** [review]

**Pull request merged**